### PR TITLE
Initial Code Mockup

### PR DESCRIPTION
**Pull request by amayer5125**
Site looks visually similar to mockup jpeg

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /vendor/
 web/.sass-cache
+web/stylesheets
```
Comment (brewcore) [inline .gitignore:3]: can you add a .gitkeep or something inside stylesheets? if the folder is missing I think some of the build tools may complain

File: web/sass/custom.scss [hunk 2]
```diff
@@ -0,0 +1,85 @@
+$light-blue: #2d83ba;
+$orange: #F09740;
+$dark-blue: #053E4C;
+$grey: #242A2F;
+$text: #FFF;
```
Comment (brewcore) [inline web/sass/custom.scss:5]: some of these vars should be in _bootstrap-variables.scss

File: .gitignore (hunk 1 above)
Comment (zachfedor) [inline .gitignore:3]: +1

File: views/bottomscripts.html [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```
Comment (zachfedor) [inline views/bottomscripts.html:1]: can we pull this in locally instead?

**Comment by amayer5125** [comment]
Should I still make these changes? or is @zachfedor working on them?

**Comment by zachfedor** [comment]
i merged it to fix it up here. i figured that was easier than forking your fork and adding a PR to your PR...

**Comment by amayer5125** [comment]
Makes sense

**Pull request merged**